### PR TITLE
[Bulk] Broken Link fixes 

### PR DESCRIPTION
**Pull request by DhurataJ**
Fix/remove broken links

### CODE REVIEW

**Comment by DhurataJ** [comment]
**Global effort to fix broken links**

@stevemunk 

The C+E Skilling team is fixing broken links on learn.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. I'd very much appreciate your timely review and feedback on the link changes I've suggested. Thanks!

File: BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md [hunk 1]
```diff
@@ -261,15 +261,15 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Kolkata|[SBSTC](http://sbstc.co.in/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md:261]: This link goes to a page that only shows: 

> Welcome to test page

File: BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md [hunk 2]
```diff
@@ -261,15 +261,15 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Kolkata|[SBSTC](http://sbstc.co.in/) 
 |Kolkata|[WBSTC](https://en.wikipedia.org/wiki/West_Bengal_Surface_Transport_Corporation) 
 |Mumbai|[BEST](https://www.bestundertaking.com/) 
-|Mumbai|[Mumbai Metro Rail Corporation Limited](https://www.mumbai-metro.com) 
+|Mumbai|[Mumbai Metro Rail Corporation Limited](https://www.mmrcl.com) 
 |Mumbai|[Mumbai Metropolitan Region Development Authority](http://mmrda.maharashtra.gov.in/home)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md:265]:
```suggestion
|Mumbai|[Mumbai Metropolitan Region Development Authority](https://mmrda.maharashtra.gov.in/home) 

```

File: BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md [hunk 3]
```diff
@@ -261,15 +261,15 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Kolkata|[SBSTC](http://sbstc.co.in/) 
 |Kolkata|[WBSTC](https://en.wikipedia.org/wiki/West_Bengal_Surface_Transport_Corporation) 
 |Mumbai|[BEST](https://www.bestundertaking.com/) 
-|Mumbai|[Mumbai Metro Rail Corporation Limited](https://www.mumbai-metro.com) 
+|Mumbai|[Mumbai Metro Rail Corporation Limited](https://www.mmrcl.com) 
 |Mumbai|[Mumbai Metropolitan Region Development Authority](http://mmrda.maharashtra.gov.in/home) 
 |Mumbai|[NMMT](http://www.nmmtonline.com/) 
 |Mumbai|[TMT](https://www.thanecity.gov.in/) 
 |New Delhi|[Delhi Integrated Multi-modal Transit System Ltd.](https://www.dimts.in/) 
 |New Delhi|[Delhi Metro Rail Corporation](http://www.delhimetrorail.com)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md:269]:
```suggestion
|New Delhi|[Delhi Metro Rail Corporation](https://www.delhimetrorail.com) 

```

File: BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md [hunk 4]
```diff
@@ -261,15 +261,15 @@ Bing Maps APIs provide transit information for the following transit agencies in
...
 |New Delhi|[Delhi Integrated Multi-modal Transit System Ltd.](https://www.dimts.in/) 
 |New Delhi|[Delhi Metro Rail Corporation](http://www.delhimetrorail.com) 
-|New Delhi|[Delhi Transport Corporation](http://www.dtc.nic.in) 
-|New Delhi|[Haryana Roadways Faridabad City Bus Service](http://hartrans.gov.in) 
-|New Delhi|[Haryana Roadways Gurgaon City Bus Service](http://hartrans.gov.in) 
+|New Delhi|[Delhi Transport Corporation](https://dtc.delhi.gov.in) 
+|New Delhi|Haryana Roadways Faridabad City Bus Service 
+|New Delhi|Haryana Roadways Gurgaon City Bus Service
 |New Delhi|[Indian Railway](https://erail.in) 
 |New Delhi|[Rapid MetroRail Gurgaon Limited](http://www.rapidmetrogurgaon.com)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md:274]:
```suggestion
|New Delhi|[Rapid MetroRail Gurgaon Limited](https://www.rapidmetrogurgaon.com) 

```

File: BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md [hunk 5]
```diff
@@ -280,7 +280,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |:--------------------|--------------|
 |Fukuoka|[福岡市営渡船](https://www.hktport.city.fukuoka.lg.jp/) 
 |Gunma|[永井運輸株式会社](https://www.nagai-unyu.net/)
-|Gunma|[Nagai Transport Co., Ltd.](http://https://www.nagai-unyu.net/)  
+|Gunma|[Nagai Transport Co., Ltd.](http://www.nagai-unyu.net/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md:283]:
```suggestion
|Gunma|[Nagai Transport Co., Ltd.](https://www.nagai-unyu.net/)  

```

File: BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md [hunk 6]
```diff
@@ -280,7 +280,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |:--------------------|--------------|
 |Fukuoka|[福岡市営渡船](https://www.hktport.city.fukuoka.lg.jp/) 
 |Gunma|[永井運輸株式会社](https://www.nagai-unyu.net/)
-|Gunma|[Nagai Transport Co., Ltd.](http://https://www.nagai-unyu.net/)  
+|Gunma|[Nagai Transport Co., Ltd.](http://www.nagai-unyu.net/)  
 |Shimada|[島田市](https://www.city.shimada.shizuoka.jp/kurashi-docs/bus.html) 
 |Yamanashi|[中央市](https://www.city.chuo.yamanashi.jp/index.html) 
 |Yamanashi|[南アルプス市](http://www.city.minami-alps.yamanashi.jp/)
```
Comment (stevemunk) [inline BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md:286]:
```suggestion
|Yamanashi|[南アルプス市](https://www.city.minami-alps.yamanashi.jp/) 

```

**Review by stevemunk — CHANGES_REQUESTED** [review]
Please use secure links where possible.

**Review by stevemunk — CHANGES_REQUESTED** [review]
I made comments on several of the `http` (not secure) links... Please make changes as needed then go through the rest `http` links and test each one in a browser after adding the 's' (`https`). If it works, change the link to `https`. If it doesn't work, make sure the original link (`http`) works and if so, leave it as it. If it doesn't work, please remove the link. Once done I will approve and merge into main.

File: BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md [hunk 7]
```diff
@@ -347,7 +347,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Taipei|[東南客運](http://www.southeastbus.com) 
 |Taipei|[板橋區公所](http://www.banqiao.ntpc.gov.tw) 
 |Taipei|[林口區公所](http://www.linkou.ntpc.gov.tw) 
-|Taipei|[桃園客運](http://www.tybus.com.tw) 
+|Taipei|[桃園客運](http://www.tybus.com) 
 |Taipei|[樹林區公所](http://www.shulin.ntpc.gov.tw) 
 |Taipei|[欣欣客運](http://www.shinbus.com.tw/) 
 |Taipei|[永和區公所](http://www.yonghe.ntpc.gov.tw)
```
[the remark this answers is not in the document]
Comment (v-emilypr) [reply BingMaps/coverage/transit-coverage/asia-australia-pacific-islands.md]: This is updated to https.

**Review by stevemunk — APPROVED** [review]

**Pull request merged**